ZeroWasteFriendly/Scenes/LibraryList/LibraryListPresenter.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  LibraryListPresenter.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 07/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+protocol LibraryListPresenter {
```

Comment:
Please reuse the presenter and controller from the other project so you have the alert in spinner already implemented